app/controllers/api/v1/articles_controller.rb

```diff
@@ -22,23 +22,29 @@ def show
 
   def create
     article = current_user.articles.create(article_params)
-
-    if article.persisted?
+    if article.persisted? && attach_image(article)
```

Comment:
What happens if a journalist creates an article without an image? Should he/she be able to do that? If not, what happens the image in the params is not present?